calico/felix/config.py

```diff
@@ -382,6 +389,28 @@ def _validate_cfg(self, final=True):
                 self.parameters["DefaultEndpointToHostAction"]
             )
 
+
+        # For non-positive time values of reporting interval we set both
+        # interval and ttl to 0 - i.e. status reporting is disabled.
+        if self.REPORTING_INTERVAL_SECS <= 0:
+            self.REPORTING_TTL_SECS = 0
+            self.REPORTING_INTERVAL_SECS = 0
+
+        # Status report time to live must be more than status reporting interval,
+        # unless TTL is default.
+        if self.REPORTING_TTL_SECS <= self.REPORTING_INTERVAL_SECS\
+                and self.REPORTING_TTL_SECS != 0:
+            raise ConfigException("Reporting TTL ({} sec) is less than or equal "
```

Comment:
I think I'd default the TTL in this case rather than raising an exception.  Imagine fat-fingering the config in a 1000-host cluster and watching your felixes die en masse.